packages/nocodb/src/run/docker.ts

```diff
@@ -18,8 +18,8 @@ server.set('view engine', 'ejs');
 process.env[`DEBUG`] = 'xc*';
 
 (async () => {
-  const httpServer = server.listen(process.env.PORT || 8080, () => {
-    console.log(`App started successfully.\nVisit -> ${Noco.dashboardUrl}`);
+  const httpServer = server.listen(process.env.PORT || 8080, async () => {
+    server.use(await Noco.init({}, httpServer, server));
```

Comment:
__Note__: Once we published we need to update the `noco-seed` repo as well to avoid duplicate log.